server/src/main/java/com/netflix/conductor/bootstrap/BootstrapUtil.java

```diff
@@ -0,0 +1,73 @@
+package com.netflix.conductor.bootstrap;
+
+import com.netflix.conductor.dao.IndexDAO;
+import com.netflix.conductor.elasticsearch.EmbeddedElasticSearch;
+import com.netflix.conductor.grpc.server.GRPCServer;
+import org.apache.log4j.PropertyConfigurator;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Optional;
+import java.util.Properties;
+
+public class BootstrapUtil {
+    public static void startEmbeddedElasticServer(Optional<EmbeddedElasticSearch> embeddedSearchInstance) {
```

Comment:
nitpicks: 
`startEmbeddedElasticServer` -> `startEmbeddedElasticsearchServer`
`embeddedSearchInstance` -> `embeddedElasticsearchInstance`

Reply:
Fixed it